sagemaker_neo_compilation_jobs/xgboost_customer_churn/xgboost_customer_churn_neo.ipynb

```diff
@@ -38,6 +38,11 @@
     "\n",
     "_This notebook was created and tested on an ml.m4.xlarge notebook instance._\n",
     "\n",
+    "_It has been tested on the following versions of important python packages:_\n",
```

Comment:
Let's (temporarily) install SM v2.:

`import sys
!{sys.executable} -m pip install -qU "sagemaker==2.??" `

By default, notebooks will use SM `v1.72.1` for now. However, there is forward-ported SDK v1 functionality. Thus, customers should be able to use SM v2 with minimal headache.

Reply:
I provide some code snippets on setup for v2 and how to roll it back at the end. You don't want to mess up someone's kernel so that other notebooks expecting v1 won't run.

Reply:
Added